keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -942,31 +861,18 @@ It can affect authentication time, but the actual latency depends on a provider
 
 ###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
 
-<!--
-Things to keep in mind include: additional in-memory state, additional
-non-trivial computations, excessive access to disks (including increased log
-volume), significant amount of data sent and/or received over network, etc.
-This through this both in small and large cases, again with respect to the
-[supported limits].
-
-[supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
--->
+No. There would be very minimal addition to the memory used by the API Server and
```

Comment:
Note the "Maybe" response to this question in https://github.com/kubernetes/enhancements/pull/4456. Does the same logic apply here?

Reply:
The "Maybe" response in #4456 is for `Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?`. Are you referring to that response for this KEP?

Reply:
My mistake, I was comparing answers between these KEPs because they have similar behavior, but got mis-aligned here.